proposals/nixpkgs-security.md

```diff
@@ -0,0 +1,278 @@
+(4) Updates are mission-critical and recovery must be assured in case of failed upgrades, e.g. A/B-style layouts like Android, embedded systems, etc.
+
+providing one of the most secure and reliable way to deploy software in production.
+
+Finally, [bootstrap]
+
+
+### What the field will learn from our work
+
+Nix puts forward a unique (indeed, we claim, revolutionary) approach to software development to begin with. One can argue that it is still somewhat obscured for various reasons, and one of them is that the core mechanisms are hidden away as an implementation detail.
+
+Clarifying concepts by formalising and documenting interfaces would enable others to build on top of, reimplement, or innovate over what Nix has to offer, and help spread its ideas beyond the currently entrenched but limited notion of a package manager and beyond the particular implementation angle the Nix ecosystem has taken.
+
+Furthermore, in all security areas we presented, most of our work is reusable and is reused or we are reusing other Linux distribution works, this is all part of a circuit where we share and we aim to do even better by abstracting what we produce as much as possible to offer it "on the shelf".
+
+### How our work will contribute to sustainability of Nix, the Nix ecosystem, and FOSS more generally
+
+Depending on the mentioned pieces will provide developer force from Nixpkgs to those upstream projects, providing maintenance and improvements.
+
+Furthermore, the security tracker will serve as a sustainability platform regarding security responses across all Nixpkgs and can also be reused for derivatives of Nixpkgs.
+
+## How will you accomplish the work? Please provide a list of deliverables with associated effort and cost of each deliverable.
+
+For the boot security chain, here is the list of deliverables:
+
+- [ ] Preparatory fundamental upstream features (systemd and more): 4w, 10K EUR.
+- [ ] NixOS's default UEFI Secure Boot images via shim and shim-review: 4w, 10K EUR (*)
+- [ ] NixOS RFC "Bootspec v2": 2w, 4K EUR
+- [ ] Lanzaboote and TPM2: 4w, 10K EUR
+- [ ] Safe updates (A/B schemas) in NixOS with systemd-boot: 2w, 5K EUR
+- [ ] Safe Nix store via integrity techniques (*-verity): 4w, 10K EUR
+- [ ] Safe NixOS via maximal reduction of attack surface through open interpreters: 4w, 10K EUR.
+- Total: 59K EUR.
+
+(*) This is dependent on the [shim-review project](https://github.com/rhboot/shim-review) owners to accept our application.
+
+Explaination:
+
+In order to make many of those features a stable and reliable reality, some work has to be done upstream
+which will benefit the whole ecosystem and community and show the talent and expertise of our ecosystem.
+
+Then, a lot of work would be performed in parallel via lanzaboote maintainers
+but new developers who joined the project because they were seduced in the value proposition of combining NixOS and those features.
```

Comment:
"but new" makes no sense here. Did you mean "by new"?

Reply:
new developers who joined after the creation of the project